pom.xml

```diff
@@ -273,7 +277,6 @@
                 <artifactId>maven-compiler-plugin</artifactId>
                 <version>3.10.1</version>
                 <configuration>
-                    <fork>true</fork>
```

Comment:
For whatever reason, maven-compiler-plugin sometimes doesn't display compiler warnings if it tries to fork a `javac` process.
So I removed it here and opted to just place it inside the `errorprone` profile, since we only need to fork when using errorprone.